### DISCUSSION TITLE
vue@3.0+ant-design-vue@2.x.  自增列表递归失效，vue@2.x + ant-v@1.7.x 是可以的

### DISCUSSION THREAD

**Discussion by yangshuwei** [alibaba/formily#1547] [Q&A 2.x] [answered]
帮忙看一下具体是什么原因？应该怎么处理？
[https://codesandbox.io/s/nostalgic-sanne-kntvx?file=/src/App.vue](url)

**Comment by yangshuwei** [comment]
https://codesandbox.io/s/nostalgic-sanne-kntvx?file=/src/App.vue

**Comment by MisicDemone** [comment, marked as answer]
vue3渲染函数的更新时非兼容的，文档里的示例是vue2的，vue3需要修改一部分写法，具体参考vue文档：https://v3.cn.vuejs.org/guide/migration/render-function-api.html#_3-x-%E8%AF%AD%E6%B3%95-2
